templates/analytics/support.html

```diff
@@ -30,6 +23,12 @@
             {{ error_message }}
         </center>
     </div>
+    {% elif 'success_message' in request.session %}
+    <div class="alert alert-success">
+        <center>
+            {{ request.session.success_message }}
+        </center>
+    </div>
```

Comment:
If we're moving the information from the session into the context (line 1115, above), we shouldn't need this hunk.  Did it not work without it?

Reply:
ah! didn't notice this earlier... made the change!